transcom/mymove
# B 19996 int

## [B-19996](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A949598)

## Summary

This task updates the TOO role to show when a domestic crating service item is Standalone in the UI. When a standalone shipment is up for approval, it will be displayed with a '- Standalone' tag on it to alert the TOO. 

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

Setup: Run db_dev_fresh to ensure db changes are in place

Note: When testing I found that many of the service items for existing data are not complete, and will cause the creation of a payment request to error out. To test a payment request with multiple service items it is easiest to just create them yourself.

1. Access the Office app
2. Login to the Prime Sim
3. Add a domestic crating service item with the standalone box checked.

4. Login as a TOO
5. Find the Move you edited in the Prime Sim
6. Note that the newly created service item show '- Standalone' next to the name
7. Approve the newly created Service Item
8. Verify it shows ' - Standalone' after the service item has been approved


### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
![Screenshot 2024-06-03 at 2 05 04 PM](https://github.com/transcom/mymove/assets/162157577/54c41c8c-42b1-4dfd-968d-435b237d5b9e)
